pkg/ssh/shell/setupCLIFromSource.sh

```diff
@@ -1,6 +1,33 @@
 #!/usr/bin/env bash
 set -e
+
+#name:TASK [install gcc]
+export DEBIAN_FRONTEND=noninteractive
+while ! gcc --version >/dev/null 2>&1; do
+    echo "GCC is not installed. Trying to install..."
+    sudo apt-get -y -o DPkg::Lock::Timeout=120 update
+    sudo apt-get -y -o DPkg::Lock::Timeout=120 install gcc
+    if [ $? -ne 0 ]; then
+        echo "Failed to install GCC. Retrying in 10 seconds..."
+        sleep 10
+    fi
+done
+#name:TASK [install go]
+install_go() {
```

Comment:
I believe we have go on ami

Reply:
Yes. 

The logic for both gcc and go checks before installing it, so if it's on AMI it will be skipped.
My thinking behind it is that AMI is here to speed things up as it pre caches dependencies like docker images and binaries we need.
But if we assume that AMI is not avaliable for whatever reason or broken, missing some pkg/image/dependency there is no critical impact on CLI except time needed for provisioniong